frontend/src/scenes/exporter/ExportViewer.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect } from 'react'
```

Comment:
I think this should go under `src/exporter`, not `src/scenes/exporter`. This is something that has its own esbuild bundle, so it should be at the same level as `toolbar` at least, meaning in `src`. The `scenes` folder normally contains app scenes... which is basically *the other* esbuild bundle.

The danger of mixing `exporter` into scenes is that someone might eventually import this from a scene, and include the `ExportViewer.scss` file and its scrollbar hack globally. Overparanoia perhaps, but that's what we're here for 😅 

The shared dashboards should probably also be moved up, but that seems out of scope 🤔